build_projects/update-dependencies/Config.cs

```diff
@@ -82,7 +82,8 @@ private static string GetDefaultUpstreamBranch()
 
         private static string GetDefaultCoreSetupVersionFragment()
         {
-            string coreSetupChannel = GetRepoMSBuildPropValue("BundledRuntimes.props", "CoreSetupChannel") ?? "master";
+            // by default, the current core-setup branch should match the current cli branch name
```

Comment:
The reason this changed is because the `CoreSetupChannel` property was removed from BundledRuntimes.props, so this code was always using `master`.  The property was removed, because we no longer need the channel of core-setup to download the runtime anymore.  We changed core-setup's publish output to be a flat list like we do for the SDK.